packages/codemods/src/transforms/styled-components-to-compiled/README.md

```diff
@@ -35,3 +35,20 @@ on your next run explicitly set the version number:
 ```bash
 npx @compiled/cli@0.6.0 --preset codemods
 ```
+
+## Gotchas
+
+`styled.div.attrs` spread properties are not supported.
+
+_Example_
+
+```
+styled.div.attrs({
```

Comment:
is this an example of what NOT to do?

Reply:
yeeeah, not sure what to put in the description here and thought maybe example would be helpful. But it feels like it's rather confusing, isn't it?